### DISCUSSION TITLE
How to configure a PowerShell 7 (Core) profile?

### DISCUSSION THREAD

**Discussion by luis-fss** [felixse/FluentTerminal#953] [Q&A] [answered]
How to configure a PowerShell 7 (Core) profile?

**Comment by hanskokx** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/1911919/130638031-4ffe670f-5a56-4ee2-b1a9-2f0cd17bdc96.png)

**Comment by luis-fss** [comment]
Thanks hanskokx, but at least on my computer the location of the Powershell executable looks something like this:
![image](https://user-images.githubusercontent.com/685069/130708256-0ed51ce5-4893-4c85-913c-31bb99b73bfd.png)

**Comment by hanskokx** [comment]
Ah, yes. You have the Windows Store version.

**Reply by luis-fss** [reply]
Got it, now I managed to configure the profile. Thank you very much.
